app/mailers/generic_mailer.rb

```diff
@@ -0,0 +1,23 @@
+# app/mailers/generic_mailer.rb
+class GenericMailer < ActionMailer::Base
+    # Set the default sender email
+    if Rails.env.development? || Rails.env.test?
```

Comment:
Set the default sender email as:
  default from: 'expertiza.mailer@gmail.com'
  
  as it is same for both the cases